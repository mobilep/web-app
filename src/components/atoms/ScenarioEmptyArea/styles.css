.ScenarioEmptyArea {
	width: 100%;
	height: 100%;
	display: flex;
	flex-direction: column;
	justify-content: center;
	align-items: center;
}

.ScenarioEmptyArea svg {
	width: 217px;
	height: 201px;
}

.ScenarioEmptyArea-icon {
	color: var(--grey5);
}

.ScenarioEmptyArea-text {
	color: var(--grey3);
}

@media (max-width: 768px) {
	.ScenarioEmptyArea {
		display: none;
	}
}
