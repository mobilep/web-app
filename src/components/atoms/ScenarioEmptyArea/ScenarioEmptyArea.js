import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { EmptyScenarioIcon } from '../../../icons';

import './styles.css';

export default class ScenarioEmptyArea extends Component {

	static propTypes = {
		text: PropTypes.string,
	}

	render () {
		return (
			<div className='ScenarioEmptyArea'>
				<EmptyScenarioIcon className='ScenarioEmptyArea-icon' />
				<div className='ScenarioEmptyArea-text'>{this.props.text}</div>
			</div>
		);
	}

}
