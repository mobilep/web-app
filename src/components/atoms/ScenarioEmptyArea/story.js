import React from 'react';
import { storiesOf } from '@storybook/react';
import { text } from '@storybook/addon-knobs';
import ScenarioEmptyArea from './ScenarioEmptyArea';

storiesOf('1. Atoms / ScenarioEmptyArea', module)
	.add('default', () => <ScenarioEmptyArea text={text('Text', 'No scenarios yet')} />);
