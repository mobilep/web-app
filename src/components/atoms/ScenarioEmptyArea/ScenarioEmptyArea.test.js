import React from 'react';
import { shallow } from 'enzyme';

import ScenarioEmptyArea from './ScenarioEmptyArea';

describe('Atom - ScenarioEmptyArea', () => {

	it('Should render', () => {
		const wrapper = shallow(<ScenarioEmptyArea text='Text' />);
		expect(wrapper).toMatchSnapshot();
	});

});
