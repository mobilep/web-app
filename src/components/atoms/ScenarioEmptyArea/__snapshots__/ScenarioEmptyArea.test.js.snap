// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom - ScenarioEmptyArea Should render 1`] = `
<div
  className="ScenarioEmptyArea"
>
  <EmptyScenarioIcon
    className="ScenarioEmptyArea-icon"
  />
  <div
    className="ScenarioEmptyArea-text"
  >
    Text
  </div>
</div>
`;
